src/main.rs

```diff
@@ -98,13 +98,6 @@ fn main() {
                     their settings",
                 ),
         )
-        .arg(
-            Arg::with_name("policies-download-dir")
-                .long("policies-download-dir")
-                .default_value(".")
-                .env("KUBEWARDEN_POLICIES_DOWNLOAD_DIR")
-                .help("Download path for the policies"),
-        )
```

Comment:
We must retain the ability to set the directory to be used as root for the downloads. This is important when running containerized, because the container user doesn't have a home directory at all:

https://github.com/kubewarden/policy-server/blob/851b8a8971c1e8d3e0063f7cb985245041440955/Dockerfile#L18-L25